pom.xml

```diff
@@ -252,7 +262,25 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-jdk14</artifactId>
         </dependency>
-        
+	<dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+        </dependency>
+	<dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jcl-over-slf4j</artifactId>
+        </dependency>
+	<dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jul-to-slf4j</artifactId>
+        </dependency>
+	<!-- reload4j is a drop-in security patch/replacement for log4j1. -->
+	<!-- (https://reload4j.qos.ch/) -->
+	<dependency>
+            <groupId>ch.qos.reload4j</groupId>
+            <artifactId>reload4j</artifactId>
+            <version>${reload4j.version}</version>
+        </dependency>
```

Comment:
**suggestion:** as the XOAI lib is the only one using this library, maybe we can move this next to them and mention within the comment to remove this direct dependency once the technical debt has been payed back? I'd also include a hint that we are not using this within our code, it's just to have it on the classpath... (like I did for the `org.slf4j.slf4j-jdk14` package.)

Reply:
Sure.
There is also an alternative of NOT doing any of this - adding these exclusion rules or including the dependency for reload - in our pom.xml; by modifying the XOAI .pom files in the local_lib... But I wanted to do this in a more obvious way, so that there is a reminder, with comments, in the main project pom file.